### ISSUE TITLE
Fix VGG16 (overfitting)

### ISSUE THREAD

**Issue by rgmantovani** [ribeiro-julio/crop-image-classification#8]
Change to weights=‘Imagenet', and analyze the original paper architecture.
Fully connected 4096 RELU
Dense 1: Softmax

**Comment by rgmantovani** [comment]
Fixed here: https://github.com/ribeiro-julio/crop-image-classification/pull/9#issue-2002917171
